src/main/java/racingcargame/domain/RacingCarGame.java

```diff
@@ -0,0 +1,30 @@
+package racingcargame.domain;
+
+import java.util.List;
+
+public class RacingCarGame {
+
+	public static final int RANDOM_NUMBER_MIN_BOUNDARY = 0;
+	public static final int RANDOM_NUMBER_MAX_BOUNDARY = 9;
+
+	private Cars racingCars;
+
+	public RacingCarGame() {
+	}
+
+	public void init(String carNames) {
+		this.racingCars = new Cars(carNames);
+	}
+
+	public Cars getRacingCars() {
+		return racingCars;
+	}
+
+	public void gameStart() {
```

Comment:
여기도 위 내용과 마찬가지로 동사부터 시작해주시면 좋을 것 같아요.

```suggestion
	public void startGame() {
```

```suggestion
        public void start()
```

Reply:
감사합니다!